pradeep-rajapaksha/laravel-coding-test-level-1
# Second Test - UI

### What does this pull request fix/change?
This PR is for implementing UIs for Events' 

### Testing steps
- Pull down this branch
- Run `composer install` and `php artisan server`
- Go to http://127.0.0.1:8000/events 
- Check that each and every functionality (create/edit/delete/view) working correctly.
